tests/unittests/Foundation/NSCharacterSetTests.m

```diff
@@ -152,71 +152,89 @@ void testCharacter(NSCharacterSet* charSet, NSString* testString) {
+    EXPECT_TRUE([set isKindOfClass:[NSMutableCharacterSet class]]);
 
     set = [NSMutableCharacterSet symbolCharacterSet];
-    ASSERT_TRUE([set isKindOfClass:[NSMutableCharacterSet class]]);
+    EXPECT_TRUE([set isKindOfClass:[NSMutableCharacterSet class]]);
 
     set = [NSMutableCharacterSet uppercaseLetterCharacterSet];
-    ASSERT_TRUE([set isKindOfClass:[NSMutableCharacterSet class]]);
+    EXPECT_TRUE([set isKindOfClass:[NSMutableCharacterSet class]]);
 
     set = [NSMutableCharacterSet whitespaceAndNewlineCharacterSet];
-    ASSERT_TRUE([set isKindOfClass:[NSMutableCharacterSet class]]);
+    EXPECT_TRUE([set isKindOfClass:[NSMutableCharacterSet class]]);
 
     set = [NSMutableCharacterSet whitespaceCharacterSet];
-    ASSERT_TRUE([set isKindOfClass:[NSMutableCharacterSet class]]);
+    EXPECT_TRUE([set isKindOfClass:[NSMutableCharacterSet class]]);
 
     set = [NSMutableCharacterSet URLFragmentAllowedCharacterSet];
-    ASSERT_TRUE([set isKindOfClass:[NSMutableCharacterSet class]]);
+    EXPECT_TRUE([set isKindOfClass:[NSMutableCharacterSet class]]);
 
     set = [NSMutableCharacterSet URLHostAllowedCharacterSet];
-    ASSERT_TRUE([set isKindOfClass:[NSMutableCharacterSet class]]);
+    EXPECT_TRUE([set isKindOfClass:[NSMutableCharacterSet class]]);
 
     set = [NSMutableCharacterSet URLPasswordAllowedCharacterSet];
-    ASSERT_TRUE([set isKindOfClass:[NSMutableCharacterSet class]]);
+    EXPECT_TRUE([set isKindOfClass:[NSMutableCharacterSet class]]);
 
     set = [NSMutableCharacterSet URLPathAllowedCharacterSet];
-    ASSERT_TRUE([set isKindOfClass:[NSMutableCharacterSet class]]);
+    EXPECT_TRUE([set isKindOfClass:[NSMutableCharacterSet class]]);
 
     set = [NSMutableCharacterSet URLQueryAllowedCharacterSet];
-    ASSERT_TRUE([set isKindOfClass:[NSMutableCharacterSet class]]);
+    EXPECT_TRUE([set isKindOfClass:[NSMutableCharacterSet class]]);
 
     set = [NSMutableCharacterSet URLUserAllowedCharacterSet];
-    ASSERT_TRUE([set isKindOfClass:[NSMutableCharacterSet class]]);
+    EXPECT_TRUE([set isKindOfClass:[NSMutableCharacterSet class]]);
 
     set = [NSMutableCharacterSet characterSetWithCharactersInString:@"ABCDEFG"];
-    ASSERT_TRUE([set isKindOfClass:[NSMutableCharacterSet class]]);
+    EXPECT_TRUE([set isKindOfClass:[NSMutableCharacterSet class]]);
 
     set = [NSMutableCharacterSet characterSetWithRange:NSMakeRange(0, 32)];
-    ASSERT_TRUE([set isKindOfClass:[NSMutableCharacterSet class]]);
+    EXPECT_TRUE([set isKindOfClass:[NSMutableCharacterSet class]]);
+}
+
+ARM_DISABLED_TEST(NSCharacterSet, ExceptionTests) {
+    NSMutableCharacterSet* set;
+    @try {
+        set = [NSMutableCharacterSet characterSetWithBitmapRepresentation:nil];
+        FAIL();
+    } @catch (NSException* e) {
+        ASSERT_OBJCEQ(e.name, @"NSInvalidArgumentException");
+    }
+
+    @
+    try {
+        set = [NSMutableCharacterSet characterSetWithContentsOfFile:nil];

```

Comment:
weird formatting. Double check that you don't have anything else weird preventing this from being correct. (clang-format uses a syntax tree form to evaluate how format which means weird formatting can sometimes be indicative of other errors).